strongbox-security/strongbox-authentication-providers/strongbox-default-authentication-provider/src/main/java/org/carlspring/strongbox/authentication/api/impl/defaulting/DefaultAuthenticator.java

```diff
@@ -1,4 +1,4 @@
-package org.carlspring.strongbox.authentication.api.impl;
+package org.carlspring.strongbox.authentication.api.impl.defaulting;
```

Comment:
How about we rename it to `org.carlspring.strongbox.authentication.api.impl.xml`? I mean... in this case it's getting the users from the XML..?